targets/lxqt

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+if [ "${TARGETNOINSTALL:-c}" = 'c' ]; then
+   if release -lt trusty; then
+       error 99 "LXQT packages are not available for Ubuntu releases prior to xenial."
```

Comment:
in this case, trusty.  
Prior to trusty, lxqt is not available at all as far as I've been able to find.